data/json/mutations/appearance_hair_styles.json

```diff
@@ -2,15 +2,14 @@
   {
     "type": "mutation",
     "id": "hair_bald",
-    "name": { "str": "Hair: none" },
-    "description": "You are bald.",
+    "name": { "str": "Hair: bald" },
+    "description": "You are bald, and have no hair.  Your head is quite shiny, however.",
```

Comment:
```suggestion
    "description": "You are bald.",
```

Reply:
Please revert the description changes.  These weren't mentioned in the pr description and they widen the scope of the changes too much.

Reply:
Are the descriptions for the new hair traits added in this PR (extremely long, floor length, etc.) alright, or should I change those to something else in addition to reverting the changes to the old hair traits?